app/src/main/java/sdp/moneyrun/RegisterPlayerActivity.java

```diff
@@ -46,9 +46,10 @@ public void onClick(View v) {
     }
     private void setRegisterFieldsForNextActivity(){
         Random random = new Random();
-        int uniquePlayerID = random.nextInt();
-        while(uniquePlayerID < 0)
-            uniquePlayerID = random.nextInt();
+//        int uniquePlayerID = random.nextInt();
```

Comment:
Unless this commented code is needed for later, I think it's a good practice to remove commented code as it can generate a lot of clutter.

Reply:
Yes, I am not sure if it will really be needed since we get the unique id from the extras passed from another activity, but in I was wondering if in the future if we have too much ( like really a lot) of unique ids that arrive, there could be a necessity for generating it with the commented code, that is why I left it but I agree it is cumbersome.

Reply:
Fair enough, we can always remove it later if we don't need it.